### ISSUE TITLE
Verhalten von GET bei leeren Listen

### ISSUE THREAD

**Issue by tabacha** [bensteffen/adfc-t30-api#9]
http://localhost:4200/t30/api/crud.php?entity=patenschaft 

```json
{"serverity":3,"message":"No data found.","code":404} 
```
Ich würde einfach eine leere Liste erwarten:

```json
[]
```
https://stackoverflow.com/questions/13366730/proper-rest-response-for-empty-table

**Comment by bensteffen** [comment]
Ja, da besteht sogar schon eine Flag "emptyResult" bei der read-Methode, die bisher nur intern verwendet wird. Damit kann man angeben welche Wert bei einem leeren Ergebnis zurückgeben wird. Standard ist null. Es fehlt noch das Interface zur API.

**Comment by bensteffen** [comment]
Ja, da besteht sogar schon eine Flag "emptyResult" bei der read-Methode, die bisher nur intern verwendet wird. Damit kann man angeben welche Wert bei einem leeren Ergebnis zurückgeben wird. Standard ist null. Es fehlt noch das Interface zur API.
